postgresql/defaults.yaml

```diff
@@ -5,10 +5,9 @@ Debian:
       - postgresql
     service:
       name: postgresql
+    cluster:
+      createcluster_bin: pg_createcluster
     config:
-      manage:
```

Comment:
The data in this file can be overriden by pillar. Can we keep the ``manage`` list?